niworkflows/anat/mni.py

```diff
@@ -48,7 +54,7 @@ class RobustMNINormalizationInputSpec(BaseInterfaceInputSpec):
                                       desc='template resolution')
     generate_report = traits.Bool(
         False, usedefault=True, desc='enable report generation')
-    html_report = File('viz-report.html', usedefault=True, desc='visual report file')
+    html_report = File('viz-report.svg', usedefault=True, desc='visual report file')
```

Comment:
I prefer the name `html_report` because it creates a strong norm for the files being html files, because we will be writing an aggregator that assumes the reports are html snippets.